topics/howtosim.rst

```diff
@@ -46,83 +47,79 @@ Parameters for ``gendisk`` are specified in a YAML file with the following field
    CMAPR                : 1                    # Radial coordinate mapping type for cylindrical grid (0:none, 1:rational fct)
    CMAPZ                : 1                    # Vertical coordinate mapping type for cylindrical grid (0:none, 1:sech, 2:power in z
    DF                   : 1                    # Use change-over from Jeans to Eddington
-   DFLAG                : 6                    # Output flags for DiskHalo
+   DFLAG                : 0                    # Output flags for DiskHalo
    DIVERGE              : 0                    # Cusp extrapolation for primary halo model
    DIVERGE2             : 0                    # Cusp extrapolation for number model
    DIVERGE_RFAC         : 1                    # Extrapolation exponent for primary mass model
    DIVERGE_RFAC2        : 1                    # Extrapolation exponent for number model
-   DMFAC                : 0.0                  # Disk mass scaling factor for spherical deprojection model
-   DR_DF                : 5                    # Width of change for to Eddington
+   DMFAC                : 1.0                  # Disk mass scaling factor for spherical deprojection model
+   DR_DF                : 1.0                  # Width of change for to Eddington
    DUMPCOEF             : false                # Dump coefficients
+   ASCALE               : XXXXX                # radial scale length for disk basis construction (e.g. 0.01)
    HSCALE               : XXXXX                # Vertical scale length for disk basis construction (e.g. 0.002)
-   LMAX                 : 6                    # Number of harmonics for Spherical SL for halo/spheroid
-   LMAXFID              : 64                   # Number of harmonics for Spherical SL for determining disk basis
+   LMAX                 : 18                   # Number of harmonics for Spherical SL for halo/spheroid
+   LMAXFID              : 128                  # Number of harmonics for Spherical SL for determining disk basis (may want to bump to 256)
    LOGR                 : true                 # Make a logarithmic coordinate mapping
    MMAX                 : 6                    # Number of azimuthal harmonics for disk basis
-   NCHEB                : 16                   # Chebyshev order for smoothing
-   NDP                  : 1
+   CHEBY                : false                # Chebyshev smoothing
```

Comment:
It probably makes sense to move these to expert parameters, as Chebyshev smoothing is generally dangerous unless using all the diagnostic files.